packages/orbit-components/src/SkipNavigation/__tests__/index.test.js

```diff
@@ -1,28 +1,46 @@
 // @flow
 import * as React from "react";
-import { shallow } from "enzyme";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
 
 import SkipNavigation from "..";
 
 describe("SkipNavigation", () => {
-  const actions = [
-    {
-      link: "https://www.kiwi.com/cz/pages/content/terms",
-      name: "Got to terms and condition",
-    },
-    {
-      name: "Add baggage",
-    },
-    {
-      name: "Reguest refund",
-    },
-  ];
-  const feedbackUrl = "https://www.kiwi.com/en/";
+  it("should have expected DOM output", () => {
+    const actions = [
+      {
+        link: "https://www.kiwi.com/cz/pages/content/terms",
+        name: "Got to terms and condition",
+      },
+      {
+        name: "Add baggage",
+      },
+      {
+        name: "Reguest refund",
+      },
+    ];
+    const feedbackUrl = "https://www.kiwi.com/en/";
+    render(<SkipNavigation actions={actions} feedbackUrl={feedbackUrl} />);
+    expect(screen.getByRole("link")).toBeInTheDocument();
+    expect(screen.getByRole("combobox")).toBeInTheDocument();
+  });
+
+  it("should toggle select on focus/blur", () => {
+    render(
+      <SkipNavigation
+        actions={[
+          {
+            name: "Add baggage",
+          },
+        ]}
+        feedbackUrl="https://www.kiwi.com/en/"
+      />,
+    );
 
-  const component = shallow(<SkipNavigation actions={actions} feedbackUrl={feedbackUrl} />);
+    const wrapper = document.querySelectorAll("div")[1];
```

Comment:
I think this element is also `container` that's being returned from `render`, could you access it that way instead?

```js
const { container } = render(
```

Reply:
i tried that and it did not work